### PR TITLE
Merge 3.8.11 Release into Joomla 4

### PR DESCRIPTION
**Pull request by wilsonge**
Please don't squash merge as this will loose commit history. Only ever merge with a merge commit

### CODE REVIEW

File: administrator/components/com_content/Model/FeaturedModel.php [hunk 1]
```diff
@@ -145,15 +145,24 @@ protected function getListQuery()
...
+			$access = ArrayHelper::toInteger($access);
+			$access = implode(',', $access);
+			$query->where('a.access IN (' . $access . ')');
+		}
 
 		// Filter by access level on categories.
 		if (!$user->authorise('core.admin'))
 		{
 			$groups = implode(',', $user->getAuthorisedViewLevels());
+			$query->where('a.access IN (' . $groups . ')');
```
Comment (brianteeman) [inline administrator/components/com_content/Model/FeaturedModel.php:165]: shouldnt the next line c.access be removed

File: build/media_source/com_associations/js/sidebyside.es5.js [hunk 2]
```diff
@@ -245,7 +246,9 @@ jQuery(document).ready(function($) {
 			var reference    = document.getElementById('reference-association');
 			var referenceId  = reference.getAttribute('data-id');
 			var languageCode = reference.getAttribute('data-language').replace(/-/, '_');
-			var title        = $(reference).contents().find('#jform_title').val();
+			var target       = document.getElementById('target-association');
```
Comment (brianteeman) [inline build/media_source/com_associations/js/sidebyside.es5.js:249]: is this correct - there is var target a few lines later ??

File: administrator/components/com_content/Model/FeaturedModel.php (hunk 1 above)
Comment (wilsonge) [reply administrator/components/com_content/Model/FeaturedModel.php:165]: It's checking you have access to the category and the article - so I don't *think* so. Just double checked and this is at least consistent with what's in staging right now https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_content/models/featured.php#L141-L147 

So if it does need fixing it's outside the scope of the merge here anyhow and should be done in staging

File: build/media_source/com_associations/js/sidebyside.es5.js (hunk 2 above)
Comment (wilsonge) [reply build/media_source/com_associations/js/sidebyside.es5.js:249]: I mean the target is used on the next line here and then overridden after being used. Which seems like a bad naming convention for all kinds of reasons. But something to be fixed again outside the scope of this PR (same thing in staging https://github.com/joomla/joomla-cms/blob/staging/media/com_associations/js/sidebyside-uncompressed.js#L290-L293)

File: administrator/components/com_content/Model/FeaturedModel.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_content/Model/FeaturedModel.php:165]: its ok I missed the query for c earlier

**Pull request merged**